website/src/css/custom.scss

```diff
@@ -736,13 +736,13 @@ img[alt="github-contribute"] {
   }
 }
 
-.docs-doc-id-current\/quickstart\/quickstart-hello,
-.docs-doc-id-current\/quickstart\/quickstart-test,
-.docs-doc-id-current\/quickstart\/quickstart-build,
-.docs-doc-id-current\/quickstart\/quickstart-publish,
-.docs-doc-id-current\/quickstart\/quickstart-build-multi,
-.docs-doc-id-current\/quickstart\/quickstart-caching,
-.docs-doc-id-current\/quickstart\/quickstart-build-dockerfile {
+.docs-doc-id-current\/quickstart\/hello,
```

Comment:
Does this mean that if we add a page to the quickstart or change a page id, this CSS requires an update?

Reply:
Yes, that is my understanding.

Reply:
Yes, it's very fragile and not the best solution at all.

There's a constraint on why I implemented it this way though: the selectors are pointing to HTML elements generated by Docusaurus, which makes it a bit trickier to do.

I added this hotfix to restore the user-facing experience only. I'm thinking of a more permanent fix, but it's not that straightforward. I'm missing something for sure, but will open a PR once I figure it out for good.